tests/e2e-vcluster/vcluster-ks-1/25-assert.yaml

```diff
@@ -0,0 +1,22 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: VerticaDB
+metadata:
+  name: v-vcluster-ks-1
+status:
+  subclusters:
+    - installCount: 3
+      addedToDBCount: 3
+      upNodeCount: 2
```

Comment:
I don't think we will able to pass this. This testcase does a stop db right after the create. But we don't have any restart capability. So, I don't think we can validate the fetch node in e2e just yet. My suggestion is to remove this file and we will validate restart, and thus fetch node, once we have more of the vclusterOps implemented.

Reply:
Thanks for this comment! I have removed `upNodeCount: 2`.